drivers/usb/udc/udc_dwc2.c

```diff
@@ -842,9 +847,58 @@ static uint8_t pull_next_ep_from_bitmap(uint32_t *bitmap)
 	}
 }
 
+static void dwc2_handle_hibernation_exit(const struct device *dev,
+					 bool remote_wakeup, bool bus_reset)
```

Comment:
Please use rwup abbreviation.
```
static void dwc2_handle_hibernation_exit(const struct device *dev,
					 const bool rwup, const bool bus_reset)
```

Reply:
Changed, but I don't quite like `rwup`. Saving few characters is not worth all the "what this rwup even means?" that will inevitably happen.